backend/plan/management/commands/trainrecommender.py

```diff
@@ -392,19 +388,18 @@ def train_recommender(
             else courses_data_from_db()
         )
 
-    if course_descriptions_path is not None:
-        course_descriptions_or_none = courses_data_from_csv(course_descriptions_path)
-    else:
-        course_descriptions_or_none = None
+    preloaded_descriptions = dict()
+    if preloaded_descriptions_path is not None:
+        preloaded_descriptions = dict(courses_data_from_csv(preloaded_descriptions_path))
 
-    if course_descriptions_path is None and verbose:
+    if preloaded_descriptions_path is None and verbose:
         print(
             "A course_description_path has not been supplied."
```

Comment:
Might want to global search for `course_description_path` and replace with `preloaded_descriptions_path` just to stay consistent.